origami.json

```diff
@@ -16,22 +16,28 @@
 	},
 	"demosDefaults": {
 		"js": "demos/src/demo.js",
-		"sass": "demos/src/demo.scss"
+		"sass": "demos/src/demo.scss",
+		"dependencies": ["o-table@^5.0.3"]
 	},
 	"demos": [
 		{
 			"name": "declarative",
 			"template": "demos/src/declarative.mustache",
 			"js": "demos/src/declarative.js",
-			"expanded": true,
```

Comment:
Now the new Registry is deployed, the `expanded` attributes can be safely removed.